index.html

```diff
@@ -0,0 +1,537 @@
+              our course from scratch to hero in computer vision area.
+            </p>
+          </div>
+          <div
+            class="
+              col-sm col-gap
+              d-flex
+              flex-row flex-sm-column
+              border
+              rounded
+              border-secondary
+              bg-w-transparent
+              m-1
+              p-2
+              align-items-center
+            "
+          >
+            <i class="fas pt-2 fa-globe"></i>
+            <a href="#" rel="noopener" class="h5 c-primary">Anywhere</a>
+            <p>
+              Study from anywhere according to your current location, everything
+              is going to be online with flexible times to complete it.
+            </p>
+          </div>
+          <div
+            class="
+              col-sm col-gap
+              d-flex
+              flex-row flex-sm-column
+              border
+              rounded
+              border-secondary
+              bg-w-transparent
+              m-1
+              p-2
+              align-items-center
+            "
+          >
+            <i class="fas pt-2 fa-book-reader"></i>
+            <a href="#" rel="noopener" class="h5 c-primary">Learn</a>
+            <p>
+              You can complete the master in your own pace, but we provide you
+              deadlines to have consideration if you want to use it.
+            </p>
+          </div>
+          <div
+            class="
+              col-sm col-gap
+              d-flex
+              flex-row flex-sm-column
+              border
+              rounded
+              border-secondary
+              bg-w-transparent
+              m-1
+              p-2
+              align-items-center
+            "
+          >
+            <i class="fas pt-2 fa-clipboard-list"></i>
+            <a href="#" rel="noopener" class="h5 c-primary">Exam</a>
+            <p>
+              In order to get a certificate, you are going to do a test and you
+              must have a grade over 70% to get your certificate.
+            </p>
+          </div>
+          <div class="col-sm-1"></div>
+          <div class="col-sm-12 py-5 d-flex justify-content-center">
+            <a
+              href="#"
+              rel="noopener"
+              class="d-none d-sm-block text-uppercase see-more text-white"
+              >SEE THE WHOLE PROGRAM</a
+            >
+            <button
+              class="
+                btn btn-lg
+                bg-primary_
+                text-white
+                py-4
+                w-75
+                d-block d-sm-none
+              "
+            >
+              SEE THE WHOLE PROGRAM
+            </button>
+          </div>
+        </div>
+      </section>
+      <section class="guests-section text-center c-dark p-2 p-sm-0">
+        <div class="row py-5">
+          <div class="col-sm-12 position-relative">
+            <h5 class="pb-2 head-title">Featured Speakers Educatia</h5>
```

Comment:
- [ ] As requested by the previous code reviewer, which is also a requirement for this, please make sure the featured speakers section is rendered using JavaScript. 
![king](https://user-images.githubusercontent.com/65852621/138378197-81935a10-8690-47a2-90de-6098aacbe2b4.png)
.